CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+13m per action. Dex-10 lasts for one engagement. A Doc cannot have more than one
+dose of DEX at a time.
+
+==Infection Wave ==
+Adrenaline Cost: 60 Points
+Duration: 	2 Actions
+Range:		A 45 degree angle centered on the Doc and extending out 15m
+    Biologic targets in the area of effect without gas masks have the poison 
+payload of your choice applied to them.
+
+==Illegal Penethylamine Agonist ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	1 Action
+Range:		Applies to the Doc only
+    For 3 turns, the doc gets a -4 to miss with guns, grenades, spells, and 
+melee weapons. They also get a +60 to reload and a +1 chance to crit.
+
+==Dex-11 ==
+Adrenaline Cost: 80 Point
+Contagion Cost: 1 Contagion
+Duration:   1 action
+Range:  Doc only
+    Your move speed increases by 6m per 2 Actions, with an absolute
+maximum of 17m per action. Dex-11 lasts for one engagement. A Doc cannot have
+more than one dose of DEX at a time.
+
+==The Good Stuff ==
+Adrenaline Cost:380 Points
+Contagion Cost: 3 Contagions
+Duration: 	2 Actions
+Range:		Effects apply only to the Doc
+    This is a fully customized brew specially grown to bond with your
+personal biochemistry. The effect lasts for 8 turns. You gain:
+* -2 to miss with melee, guns, thrown weapons, or Procedures
+* an additional action per turn
+* an additional temporary 40 hp
+* an additional 2m move speed per action. 
+
+== Whateverectomy ==
+Adrendaline Cost: 140
+Needle Cost: 3 Needles
+Duration 2 Actions
+Range: 5m
+    Oh, that looks infected. It's probably gonna have to come out. You can sign
+the consent forms afterwards. Well, then again, maybe you can't. The target
+Makes a DC 100 Reflex Save. On success, the target moves up to 2m in the 
+direction of their choosing and is prone. On fail, the target rolls a D10  and 
+divide the roll by 2 to see what they lose. 
+1. Their Arm. 
+2. Their Leg
+3. Their Eye
+4. Their spleen
+5. Their Lymph Node
+
+    Their Arm: The target loses an arm. Using a two handed weapon gives them a 
+-4 to hit. Damage with melee weapons is halved. You gain 3 Contagions. They take
+50 damage and have 3 stacks of Bleeding. See the "Conditions" section of Basic
+Rules.
+
+    Their Leg: The target loses a leg. Their move speed is halved round down and
+getting up after being crouched or prone takes an extra action. They take 60
+damage and receive 3 stacks of Bleeding. See the "Conditions" section of Basic
+Rules. You gain 3 Contagions. 
+
+    Their Eye: The target loses one of their eyes. They take a permanent -2 to
+their Perception and receive 60 damage. Their to hit score is reduced by 2. 
+You gain 1 Contagion.
+
+    Their Spleen: The target loses their spleen. They take 60 damage and their
+Fortitude is permanently reduced by 2. You gain 2 Contagions.
+
+    Their Lymph Node: The target loses their lymph node. They lose 40 health and
+120 Nanites. You gain 2 Contagions.
```

Comment:
>and 120 Nanites

Can we add an "up to" or something? To be crystal clear that the player won't get negative Nanites if they have a pool of 80, for example.

Reply:
Absolutely! Will do! good catch.